Cannot upgrade `camunda-bpmn-js` via `npm`

__Describe the Bug__

Running `npm update --save` after I've updated `camunda-bpmn-js` to 0.15.2 leads to:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: project@0.0.0
npm ERR! Found: @bpmn-io/properties-panel@0.15.0
npm ERR! node_modules/@bpmn-io/properties-panel
npm ERR!   peer @bpmn-io/properties-panel@"0.15.x" from camunda-bpmn-js@0.15.2
npm ERR!   node_modules/camunda-bpmn-js
npm ERR!     dev camunda-bpmn-js@"0.15.2" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @bpmn-io/properties-panel@"0.16.x" from bpmn-js-properties-panel@1.3.0
npm ERR! node_modules/bpmn-js-properties-panel
npm ERR!   peer bpmn-js-properties-panel@"1.x" from camunda-bpmn-js@0.15.2
npm ERR!   node_modules/camunda-bpmn-js
npm ERR!     dev camunda-bpmn-js@"0.15.2" from the root project
```

Actually leaving it to version 0.15.1 and running the same command again leads to an error:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: project@0.0.0
npm ERR! Found: @bpmn-io/properties-panel@0.13.2
npm ERR! node_modules/@bpmn-io/properties-panel
npm ERR!   peer @bpmn-io/properties-panel@"0.13.x" from camunda-bpmn-js@0.15.1
npm ERR!   node_modules/camunda-bpmn-js
npm ERR!     dev camunda-bpmn-js@"0.15.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @bpmn-io/properties-panel@"0.16.x" from bpmn-js-properties-panel@1.3.0
npm ERR! node_modules/bpmn-js-properties-panel
npm ERR!   peer bpmn-js-properties-panel@"1.x" from camunda-bpmn-js@0.15.1
npm ERR!   node_modules/camunda-bpmn-js
npm ERR!     dev camunda-bpmn-js@"0.15.1" from the root project
```

I have to pin `bpmn-js-properties-panel` to `1.2.0` to overcome this issue. But then I can't update `camunda-bpmn-js`. It's a mess. :smile: 

__Steps to Reproduce__

Create a new empty directory, `cd` into it and run `npm install camunda-bpmn-js`. It fails.

__Expected Behavior__

To not have versioning conflicts like that.

__Environment__

- OS: Linux
- Modeler version: 0.15.2

--- a/package.json
+++ b/package.json
@@ -98,7 +98,7 @@
     "webpack": "^5.20.1"
   },
   "peerDependencies": {
-    "@bpmn-io/properties-panel": "0.15.x",
+    "@bpmn-io/properties-panel": "0.16.x",
     "bpmn-js-properties-panel": "1.x"
   },
   "files": [

